python/neuroglancer/pipeline/storage.py

```diff
@@ -220,6 +223,10 @@ def get_file(self, file_path):
         except IOError:
             return None, False
 
+    def exists(self, file_path):
+        path = self.get_path_to_file(file_path)
+        return os.path.exists(path) or os.path.exists(path + '.gz')
```

Comment:
Nah, we need an easy way to refer to gzipped files. See #63 The alternatives are messier.